gateway/dataimpl/server_v1/errorhandler.go

```diff
@@ -145,6 +145,29 @@ func (e ErrorHandler) NewBucketExistsStatus(baseErr error, bucketName string) *s
 	return st
 }
 
+func (e ErrorHandler) NewBucketInvalidArgStatus(baseErr error, msg string, bucketName string) *status.Status {
+	if msg == "" {
```

Comment:
If msg is empty I think we should probably just make it "invalid argument"?

Reply:
Changed, I thought that we may as well specify that the invalid argument occurred when trying to create a bucket

Reply:
Being really nitpicky here but I don't think we can say "when creating bucket" as this could be used on an update.

Reply:
Done!